packages/ai-commands/src/sql/cron.ts

```diff
@@ -0,0 +1,96 @@
+import type OpenAI from 'openai'
+import { ContextLengthError } from '../errors'
+import { jsonrepair } from 'jsonrepair'
+
+// Responds to a natural language request for cron syntax.
+export async function chatCron(openai: OpenAI, prompt: string) {
```

Comment:
Might be worth renaming this to something like `generateCron()`. Currently we use the `chat___` convention for conversation-style AI that are multi-turn.